api/app/src/external/fhir/organization/upsert-organization.ts

```diff
@@ -0,0 +1,13 @@
+import { Organization } from "@medplum/fhirtypes";
+import { api } from "../api";
+import { capture } from "../../../shared/notifications";
+
+export const upsertOrgToFHIRServer = async (organization: Organization) => {
+  try {
+    return await api.updateResource(organization);
+  } catch (err) {
+    capture.error(err, {
+      extra: { context: `fhir.add.organization` },
+    });
+  }
```

Comment:
I see that in this case this is an additional call to store the Org on the FHIR server - beyond doing that on Postgres, so I think its nice to have an error handling to indicate a potential error is specific with the FHIR server upsert, not our BE's. (this shouldn't be needed if we had proper source maps on Sentry, though)

But I think we should be rethrowing errors, no?

Reply:
I changed all of these to not have a try catch so the asynchandler handles it